src/FACTFinder/Adapter/Search.php

```diff
@@ -5,6 +5,10 @@
 
 class Search extends AbstractAdapter
 {
+    
+    const STATUS_NO_ARTNRSEARCH = "noArticleNumberSearch";
```

Comment:
The implemented solution has no method to check whether an unsuccessful article number search has been made. I recommend to use an enum type with the methods `isArticleNumberResultFound()`, `isNoArticleNumberResultFound()` and `isNoArticleNumberSearch()`. They will allow a differentiated output depending on the article number search state on the result or no-hit page.